Speed-up geotif creation

kikislater opened the pull request.

kikislater committed:
Speed-up geotif creation

spifftek70 commented:
<img width="427" alt="image" src="https://github.com/user-attachments/assets/c09ca4c9-0bd8-4edf-ad01-5285a8d102e4">

Tried it and I don't see any difference.

kikislater commented:
If you have GDAL_NUM_THREADS in your environment variables or in the gdal configuration file, you won't notice it... 
I tested with 289 images, it took 45 min without this PR in COG. I added num_threads=‘all_cpus’ and changed the compression to jpeg and that took 11 minutes in COG too. Deflate is also less efficient, but I'm going to do another PR on compression, it's interesting to have compression other than deflate for RGB drone images from my point of view (for multispectral images deflate is ok) and to have 58mo tif instead of the original 11 mo...

spifftek70 commented:
I did not run with COG selected.... testing again. Add notes and testing parameters in the future so I know what to look for. And thanks for contributing!

spifftek70 commented:
<img width="200" alt="image" src="https://github.com/user-attachments/assets/f9e5f6b2-d573-4923-a1f9-2df1d2ce6f12">


{'EPSG': 4326, 'declination': True, 'COG': True} vs {'EPSG': 4326, 'declination': True}
271 GeoTiffs created.  Six seconds faster.

kikislater commented:
What is the output of 
```printenv | grep GDAL``` 
in bash
